app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -57,6 +62,8 @@ import java.util.Locale
  * @param selectedWallpaper The currently selected wallpaper.
  * @param onSelectWallpaper Callback for when a new wallpaper is selected.
  * @param onViewWallpaper Callback for when the view action is clicked from snackbar.
+ * @param tapLogoSwitchChecked Enabled state for switch controlling taps to change wallpaper.
+ * @param tapLogoSwitchOnCheckedChange Callback for when state of above switch is updated.
```

Comment:
```suggestion
 * @param onCheckedChange Callback for when state of above switch is updated.
```
Can we use the same `onXXXX` wording for our Compose callbacks here?

Reply:
Good call, updated